deploy/webhook.yaml

```diff
@@ -0,0 +1,265 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: service-apis
+  labels:
+    name: service-apis
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: service-apis-admission
+webhooks:
+  - name: validate.networking.x-k8s.io
+    matchPolicy: Equivalent
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: [ "networking.x-k8s.io" ]
+        apiVersions: [ "v1alpha1" ]
+        resources: [ "httproutes" ]
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: service-apis-admission
+        namespace: service-apis
```

Comment:
Should this be `service-apis-system` or `kube-system` namespace?

Reply:
I've heard that many k8s maintainers have dislike the idea of sticking things in the kube-system and it has started to get out of control. IMO we should deploy it to its own namespace much like ingress-nginx does. I'm happy to be convinced otherwise.

Reply:
I agree that a unique namespace for this is a good default - I like `service-apis-system`.